definitions/npm/sentry-expo_v1.11.x/flow_v0.83.x-/sentry-expo_v1.11.x.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * sentry-expo primarily delegates to the `react-native-sentry` package.
+ * Definitions pulled from https://github.com/getsentry/react-native-sentry/blob/master/lib/Sentry.d.ts.
+ */
+declare module "sentry-expo" {
+  declare type SentryBreadcrumbType = "navigation" | "http";
+
+  declare export type SentryBreadcrumb = {
+    message?: string;
+    category?: string;
+    level?: SentrySeverity;
+    data?: {};
+    type?: SentryBreadcrumbType;
+  }
+
+  declare export type SentrySeverity = "fatal" | "error" | "warning" | "info" | "debug" | "critical"
+
+  declare type SentryLog = {
+    None: 0,
+    Error: 1,
+    Debug: 2,
+    Verbose: 3
+  }
+
+  declare type SentryOptions = {
+    logLevel?: SentryLog;
+    instrument?: boolean;
+    disableNativeIntegration?: boolean;
+    ignoreModulesExclude?: [string];
+    ignoreModulesInclude?: [string];
+  }
+
+  declare export default class Sentry {
+    install(): void;
+
+    static config(dsn: string, options?: SentryOptions): Sentry;
+
+    static isNativeClientAvailable(): boolean;
+
+    static crash(): void;
+
+    static nativeCrash(): void;
+
+    static setEventSentSuccessfully(callback: Function): void;
```

Comment:
Maybe `() => void` instead of `Function`?

Reply:
👍